frontend/packages/dev-console/src/components/guided-tour/index.ts

```diff
@@ -11,50 +10,47 @@ const getSelector = (id: string): string => `[data-tour-id="${id}"]`;
 
 export const getGuidedTour = (): TourDataType => ({
   intro: {
-    heading: i18n.t('devconsole~Welcome to the Developer Perspective!'),
+    heading: '%devconsole~Welcome to the Developer Perspective!%',
```

Comment:
in plugin we even have comments along with `%` , is that not needed

```suggestion
     // t('devconsole~Welcome to the Developer Perspective!')
    heading: '%devconsole~Welcome to the Developer Perspective!%',
```